kiwi/volume_manager/base.py

```diff
@@ -61,18 +61,16 @@ def __init__(
     ) -> None:
 
         self.temp_directories: List[Temporary] = []
-        # all volumes are combined into one mountpoint. This is
-        # needed at sync_data time. How to mount the volumes is
-        # special to the volume management class
+        #: all volumes are combined into one mountpoint. This is
+        #: needed at sync_data time. How to mount the volumes is
+        #: special to the volume management class
         #: root mountpoint for volumes
```

Comment:
The last sentence of this documentation string doesn't fit into the new text anymore
```suggestion
```

Reply:
yep fixed